### PR TITLE
HDR Game mode support

### PR DESCRIPTION
**Pull request by akhilbhas**
Add HDR Game mode support for L1 and L2 tests

### CODE REVIEW

File: docs/pages/tv-settings_L2_Low_Level_Test_Spec.md [hunk 1]
```diff
@@ -1044,7 +1044,7 @@ If user chose to run the test in interactive mode, then the test case has to be
 | Variation / Steps | Description | Test Data | Expected Result | Notes|
 | -- | --------- | ---------- | -------------- | ----- |
 | 01 | Initialize the TV using TvInit | None | tvERROR_NONE | Should be successful |
-| 02 | Set the TV Dolby Vision mode using SetTVDolbyVisionMode | dolbyMode = tvDolbyMode_Dark to tvHLGMode_Reserved3 | tvERROR_NONE | Should be successful |
+| 02 | Set the TV Dolby Vision mode using SetTVDolbyVisionMode | dolbyMode = tvDolbyMode_Dark to tvHLGMode_Game | tvERROR_NONE | Should be successful |
```
Comment (Anbukannadhasan) [inline docs/pages/tv-settings_L2_Low_Level_Test_Spec.md:1047]: Can you please correct the sentence for the test data as below?

dolbyMode = value retrieved from GetTVSupportedDolbyVisionModes.

**Review by Anbukannadhasan — COMMENTED** [review]
Added review comment

File: docs/pages/tv-settings_L2_Low_Level_Test_Spec.md (hunk 1 above)
Comment (akhilbhas) [reply docs/pages/tv-settings_L2_Low_Level_Test_Spec.md]: updated the sentence.

**Review by Anbukannadhasan — APPROVED** [review]
approved

**Pull request merged**